Container/Container.cs

```diff
@@ -1,57 +1,33 @@
-﻿using System.ComponentModel.DataAnnotations;
-
-namespace Container
+﻿namespace Container
 {
     public class Container
     {
-        List<int> container = new List<int>();
-        public int maxLength = 5;
+        private readonly List<int> _container = new(capacity: 5);
+
+        public int MaxLength { get; } = 5;
 
-        public void TryAdd(int value)
+        public bool TryAdd(int value)
         {
-            if (container.Count < maxLength+1)
-            {
-                container.Add(value);
-            }
-            else
+            if (_container.Count == MaxLength)
             {
-                Console.WriteLine("Operation not possible.");
```

Comment:
Zakładając, że jest to biblioteka, którą mogłabyć chcieć gdzieś udostępnić, pisanie do konsoli nie jest dobrą praktyką. Ktoś korzystający z Twojej biblioteki może nie chcieć widzieć takich komunikatów. Lepszą opcją jest tu zwracanie informacji o tym, czy dodanie się udało czy nie. Dlatego dodałem `boolean` na wyjściu metody.